src/Installer/PluginInstaller.php

```diff
@@ -116,7 +116,7 @@ public static function postAutoloadDump(Event $event)
         $composer = $event->getComposer();
         $config = $composer->getConfig();
 
-        $vendorDir = $config->get('vendor-dir');
+        $vendorDir = realpath($config->get('vendor-dir'));
```

Comment:
@ADmad I totally forgot you had mentioned it but yes - composer's `$config->get()` does not promise an absolute path which the installer needs.